api/src/main/data/bt_language_names.xml

```diff
@@ -36,6 +36,39 @@ QUESTIONS?
 <iso639-1>sq</iso639-1>
 <script>Latn</script>
 <symbol>ALBANIAN</symbol>
+<aux_name0>Albanian (macrolanguage)</aux_name0>
+</language>
+<language>
+<id>136</id>
+<name>Albanian, Arbereshe</name>
```

Comment:
Why do some language names get rearranged with commas and some don’t?

Reply:
I can't really find any concrete reasoning.  Example "Korea, South" vs "South Ndebele."  Why would these not follow same convention.  There were not many multiword languages to go off previously though.